components/buttons/buttons.css

```diff
@@ -0,0 +1,153 @@
+.btn{
+    border-radius: 0.3rem;
+    cursor: pointer;
+    
+}
+
+.btn-padding{
+    padding: 0.5rem 2rem;
+}
+
+.btn-solid{
+    border: none;
+}
+
+.btn-outline{
+    background-color: var(--white-color);
+    border: 1px solid;
+}
+
+.btn-hover:hover{
+    background-color: var(--darkblue-color);
+}
+
+.outline-hover:hover{
+    border-color: var(--primary-color);
+    color: var(--primary-color);
+
+}
+
+.btn-active:active{
+    background-color: var(--darkred-color);
+}
+
+.outline-active:active{
+    background-color: var(--basered-color);
+    color: var(--white-color);
+}
+
+
+.btn-disable:hover{
+    cursor: no-drop;
+    background-color: var(--lightyellow-color);
+}
+
+
+/* add minus btn -------------- */
+
+.flex-btns{
+    display: flex;
+    align-items: center;
+    gap: 20px;
+}
+
+.btn-round{
+    border-radius: 50%;
+    display: inline-flex;
+    justify-content: center;
+    align-items: center;
+    height: 2.5rem;
+    width: 2.5rem;
+    border: 1px solid var(--lightgray-color);
+    box-shadow: 1px 1px 3px 1px var(--gray-color);
+    background-color: var(--white-color);
+}
+
+.btn-round:active{
+    box-shadow: 0px 0px 3px 0 var(--gray-color);
+}
+
+/* ICON BUTTONS----------------------- */
+
+.btn-icon{
+    border: none;
+    border-radius: 50%;
+    display: inline-flex;
+    justify-content: center;
+    align-items: center;
+    height: 2.5rem;
```

Comment:
Try to use those values also as var, it will be helpful in future projects

Reply:
sure...thanks